Dockerfile

```diff
@@ -18,8 +18,8 @@ from ruby:2.6
 label maintainer="Jon Leech <devrel@oddhack.org>"
 
 # Install standard Debian packages
-run apt update -qq
-run apt install -y -qq \
+# TODO add --no-install-recommends
+run apt-get update -qq && apt-get install -y -qq \
```

Comment:
AFAICT apt is a complete replacement for apt-get and I hate extra typing :-)

Reply:
not for scripts - if you leave apt in there, it will curse at you saying that apt doesn't have a stable CLI interface.

Reply:
I guess. The scripts are running against what's basically a static base image though.

Reply:
that's true. I just don't like seeing warnings. You're welcome to reject this part of the change.

(FWIW, though, it's good to have the apt update, apt install, and apt clean in a single "RUN" step, to minimize the number and size of the intermediate layers, that was the main point of this change.)